democracy_club/templates/about/index.html

```diff
@@ -54,37 +47,19 @@
-our amazing volunteers, we aggregate this data and publish it for use by anyone.
-We are committed to open data. We believe that making the data that we collect
-available to everyone is the best way of empowering the most people to vote.
+Much of our work involves collecting data from the UK's approximately 350 local authorities, either by communicating with them directly or taking information from their websites using our unique infrastructure of automated 'scrapers'. Working with our amazing volunteers, we aggregate this data and publish it for use by anyone. 
 
-We believe that we make our greatest impact when we work in partnership with
-other organisations. In the past we've collaborated with other civic organisations,
-charities, and media to reach millions more people with
-information and research.
+We are committed to open data. We believe that making the data that we collect available to everyone is the best way of empowering the most people to vote.
 
 **[Meet the team](/about/team/)**
 
 ## Community and volunteers
 
-Democracy Club's election candidate database is compiled and maintained by volunteers.
-Our volunteers are non-partisan, either interested in democracy, open source technology,
-or both. Around elections, we're supported by a network of supporters who crowdsource
-publicly available information like candidate manifestos. We also host an online
-community of 1,000 members and a mailing list with tens of thousands of subscribers.
-
-The focus of this work occurs every year in March and April, when candidate lists
-are published for the May local elections. We rely on our volunteers to collect
-these (mainly PDF) documents and enter the information into our database, with a
-little help from our automation software. In 2022, which was a relatively quiet
-election, 1,500 volunteers made 64,000 edits to our database, collecting details on
-over 21,000 individual candidates.
-
-Want to take part? To get involved, [head over to our database](https://candidates.democracyclub.org.uk/volunteer/) to create an account.
-We also recommend joining our [mailing list](https://mailinglist.democracyclub.org.uk/subscription/form) to keep updated.
+Democracy Club's candidate database is compiled and maintained by volunteers. Our volunteers are non-partisan, either interested in democracy, open source technology, or both. 
+
+The focus of this work occurs every year in March and April, when candidate lists are published for the May local elections. We rely on our volunteers to collect these (mainly PDF) documents and enter the information into [our database](https://candidates.democracyclub.org.uk/), with a little help from our automation software. We then enrich this data with extra information like contact details or manifestos. In 2022, which was a relatively quiet election, 1,500 volunteers made 64,000 edits to our database, collecting details on over 21,000 individual candidates.
```

Comment:
what about in a really _busy_ year?

Reply:
Yeah, I left this becuase I was thinking about replacing it with the GE. Leave it for now and I will update it when [this](https://app.asana.com/0/1204880927741389/1208857795878525) is done.